lib/xendit_api/api/virtual_account.rb

```diff
@@ -10,8 +10,9 @@ def create(params)
         params[:is_closed] = true if params[:is_closed].nil?
         params[:is_single_use] = true if params[:is_single_use].nil?
         params[:expected_amount] = params[:amount] unless params[:amount].nil?
-
-        response = client.post(PATH, params)
+        headers = {}
+        headers['for-user-id'] = params[:for_user_id] unless params[:for_user_id].nil?
```

Comment:
sorry for the late comment, I think the wrapper should not be responsible for handling the headers condition.
just pass the headers to the post method, and let the client handle the value of the header.

Reply:
and also the source of headers should not come from the params